src/site/content/en/blog/generic-sensor/index.md

```diff
@@ -0,0 +1,478 @@
+interface simplifies the implementation and specification process for the
+concrete sensor classes. For instance, take a look at the
+[Gyroscope](https://w3c.github.io/gyroscope/#gyroscope-interface)
+class, it is super tiny! The core functionality is specified by the base
+interface, and Gyroscope merely extends it with three attributes representing
+angular velocity.
+
+Typically a concrete sensor class represents an actual sensor on the platform,
+e.g., accelerometer or gyroscope. However, in some cases, implementation of a
+sensor class [fuses](https://w3c.github.io/sensors/#sensor-fusion) data from
+several platform sensors and exposes the result in a convenient way to the user.
+For example, the
+[AbsoluteOrientation](https://www.w3.org/TR/orientation-sensor/#absoluteorientationsensor)
+sensor provides a ready-to-use 4×4 rotation matrix based on the data obtained
+from the accelerometer, gyroscope, and magnetometer.
+
+You might think that the web platform already provides sensor data and you are
+absolutely right! For instance, [DeviceMotion](https://developer.mozilla.org/en-US/docs/Web/API/DeviceMotionEvent)
+and [DeviceOrientation](https://developer.mozilla.org/en-US/docs/Web/API/DeviceOrientationEvent)
+events expose motion sensor data. So why do we need new API?
+
+Comparing to the existing interfaces, Generic Sensor API provides great
+number of advantages:
+
+- Generic Sensor API is a sensor framework that can be easily extended
+  with new sensor classes and each of these classes will keep the generic
+  interface. The client code written for one sensor type can be
+  reused for another one with very few modifications!
+- You can configure the sensor, for example, set the sampling frequency suitable for your
+  application needs.
+- You can detect whether a sensor is available on the platform.
+- Sensor readings have high precision timestamps, enabling better
+  synchronization with other activities in your application.
+- Sensor data models and coordinate systems are clearly defined, allowing
+  browser vendors to implement interoperable solutions.
+- The Generic Sensor based interfaces are not bound to the DOM (neither
+  `navigator` nor `window` objects), and it opens up future opportunities of using
+  the same API within service workers or implementing Generic Sensor API in headless
+  JS runtimes, for instance, on embedded devices.
+- [Security and privacy](#privacy-and-security) aspects are the top priority
+  for the Generic Sensor API and provide much better security level compared to
+  older sensor APIs. There is integration with Permissions API.
+- Automatic [synchronization with screen coordinates](#synchronization-with-screen-coordinates) is
+  available for `Accelerometer`, `Gyroscope`, `LinearAccelerationSensor`, `AbsoluteOrientationSensor`,
+  `RelativeOrientationSensor` and `Magnetometer`.
+
+## Generic Sensor APIs in Chrome {: #generic-sensor-api-in-chrome }
```

Comment:
Ah, OK. This was written for developers.google.com/web. It would be good to rephrase this to focus on cross-browser compatibility rather than Chrome only.

Reply:
Fixed.